app/controllers/IncomeController.scala

```diff
@@ -110,34 +125,49 @@ class IncomeController @Inject()(personService: PersonService,
           journeyCacheService.collectedValues(Seq(UpdateIncome_PayToDateKey, UpdateIncome_IdKey, UpdateIncome_NameKey), Seq(UpdateIncome_DateKey)) flatMap tupled {
             (mandatorySeq, optionalSeq) => {
               val date = optionalSeq.head.map(date => LocalDate.parse(date))
-              EditIncomeForm.bind(mandatorySeq(2), BigDecimal(mandatorySeq.head), date).fold(
-                formWithErrors => {
+              val employerName = mandatorySeq(2)
```

Comment:
Like that you have clarified what mandatorySeq(2) is. I think we could do it for pay as well. 
`val payToDate = BigDecimal(mandatorySeq.head)`